engine/src/physics/solver/penetration_constraint/physics_constants_pc.hpp

```diff
@@ -0,0 +1,36 @@
+/// @file
+/// @brief Resource @ref cubos::engine::MagiConfig.
+/// @ingroup physics-plugin
+
+#pragma once
+
+#include <cubos/core/reflection/reflect.hpp>
+
+#include <cubos/engine/api.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Resource which allows configuration over magic numbers in the solver plugin.
+    /// @ingroup physics-plugin
+    struct PhysicsConstantsPC
+    {
+        CUBOS_REFLECT;
+
+        float biasMax = -4.0F;
+
+        float contactHertzMin = 30.0F;
+
+        float kNormal = 0.0F;
+
+        float kFriction = 0.0F;
+
+        float cmpTangentLenSq = 1e-6F * 1e-6F;
+
+        float cmpRestitution = 0.0F;
+
+        float cmpNormalSpeed = -0.01F;
+
+        float cmpNormalImpulse = 0.0F;
```

Comment:
Same thing as before here, they are all minimum values. The kNormal and KFriction should also be specified as min.